test/helpers/fixture_set.rb

```diff
@@ -126,6 +126,15 @@ def assert_service_account_present(name)
       assert desired.present?, "Service account #{name} does not exist"
     end
 
+    def assert_daemon_set_present(name)
+      found = false
+      daemon_sets = v1beta1_kubeclient.get_daemon_sets(namespace: namespace, label_selector: "app=#{name}")
+      daemon_sets.each do |ds|
+        found = true if ds.metadata.name == name
+      end
+      assert found
```

Comment:
We should also have an availability assertion like we do for deployments and replicasets.